scenes/game/game.gd

```diff
@@ -1433,8 +1453,12 @@ func _on_advance_turn():
 		change_ui_state(UIState.UIState_PickTurnAction, UISubState.UISubState_None)
 		deselect_all_cards()
 		close_popout()
+		if player_clock_remaining < minimum_time_per_turn:
+			player_clock_remaining = minimum_time_per_turn
```

Comment:
I was expecting to reset the time whenever you had to make a decision instead of per turn.
You could imagine it is your turn and you do a strike that ends up having like 5 different decisions as part of boosts/abilities and that would make it extremely tight to fit it all in.
I'm not sure where would be best to actually reset the time though.
Maybe whenever current_clock_user actually gets assigned to one of the 2 players?